PhysicsTools/Heppy/interface/JetUtils.h

```diff
@@ -0,0 +1,18 @@
+#ifndef PhysicsTools_Heppy_JetUtils_h
+#define PhysicsTools_Heppy_JetUtils_h
+
+#include "DataFormats/Common/interface/Ptr.h"
+#include "DataFormats/PatCandidates/interface/Jet.h"
+
+namespace heppy{
+
+  struct JetUtils{
+
+    static const pat::Jet 
+    copyJet(const edm::ProductID& id, const pat::Jet& ijet, unsigned long key);
```

Comment:
By hiding the edm::Ptr, you mean in a new pat::Jet constructor? (not sure about what you have in mind, as it is already in the C++ function)